### PR TITLE
Ruby group capstone 🥇 

### PR DESCRIPTION
**Pull request by ismailmunyentwari9**
# What We have done in all this capstone project as a group 🌴 

- [ ] Create a Book class in a separate .rb file.
- [ ] Create a Label class with an association to the Item class.
- [ ] Create MusicAlbum class in a separate .rb file.
- [ ] Create Genre class with an association to the Item class
- [ ] Create a Game class in a separate .rb file.
- [ ] Create an Author class with an association to the Item class.

# 🧠 Skills we gained In this capstone project as agroup 🛹 

- [ ] Insert and query data in SQL.
- [ ] Use primary key & foreign key mechanisms for joining tables.
- [ ] Understand the different types of relationships between tables.
- [ ] Query multiple tables.
- [ ] Prepare complex queries that answer analytical questions.
- [ ] Run a program using the command line.
- [ ] Use Ruby syntax for basic programming operations.
- [ ] Apply Ruby best practices and language style guides in code.

etc.....

### CODE REVIEW

File: tests/author_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    end
+  end
+
+  describe '#add_item' do
+    let(:item) { double('Item') }
+
+    it 'adds the author to the item\'s list of authors' do
+      expect(item).to receive(:add_author).with(author)
+      author.add_item(item)
+    end
```
Comment (elisha2kyakpo1) [inline tests/author_spec.rb]: - [ ] Kindly fix this failing test, all the tests should pass(green).

tests

File: README.md [hunk 2]
```diff
@@ -1 +1,164 @@
...
+
+- Add a Book and associated label👌
+- Add a music album with genre 💯
+- Add a Games with authors👌
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://drive.google.com/file/d/1ONBwtug8HgDCt2sMT0WAMFIaIS594XkS/view?usp=sharing)
+
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] Great job recording a video, but we have to follow some instructions for the video, such as the length, a piece of code you are proud of, etc. I have highlighted the requirements to follow for the video in the image below. 

video

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Tests passing👏
👏 The app works as expected, adding items and preserving data.

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly rename the folder containing tests, from tests to spec because when I run RSpec when testing, RSpec will know which folder contains tests.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ In this project, We built a console app that stores books, label, music albums,
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://drive.google.com/file/d/1ONBwtug8HgDCt2sMT0WAMFIaIS594XkS/view?usp=sharing)
+- [Live Demo Link](https://drive.google.com/file/d/1L9hna1WEDODUbf6dDOwctaoWvLJZZ6P-/view?usp=sharing)
```
Comment (TadesseDev) [inline README.md]: - [Optional] Dear Team, I have seen you have mentioned that your third partner of yours is not available for the video presentation. In such a case, please reach out to student success and have their permission to don the presentation alone.

**Review by TadesseDev — APPROVED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (Newtayo) [reply README.md]: @TadesseDev This is noted. Thank you for your feedback

**Pull request merged**